docs/application/tizen-studio/web-tools/config-editor.md

```diff
@@ -1758,6 +1860,89 @@ The following sections show additional configuration elements used in the `confi
 	</tbody>
 </table>
 
+<a name="ww_launch_screen"></a>
+### Tizen Launch Screen
+
+<table>
+	<thead>
+		<tr>
+			<th>&lt;tizen:launch_screen/&gt; element</th>
+		</tr>
+		</thead>
+		<tbody>
+		<tr>
+			<td>Used to indicate the splash screen.
+			<p><strong>Occurrences:</strong></p>
+			<ul>
+				<li>0 or 1</li>
+			</ul>
+			<p><strong>Expected children:</strong></p>
+			<ul>
+				<li><code>tizen:ls_default</code>
+				<p>Used to indicate the default of the splash screen.</p>
```

Comment:
Suggestion: Can we add value?
Used to indicate the default value of the splash screen.

Reply:
> Used to indicate the default value of the splash screen.
done